Parent/Parent/Assignments/AssignmentDetailsViewController.swift

```diff
@@ -235,3 +219,18 @@ class AssignmentDetailsViewController: UIViewController, CoreWebViewLinkDelegate
         env.router.show(compose, from: self, options: .modal(embedInNav: true), analyticsRoute: "/conversations/compose")
     }
 }
+
+extension AssignmentDetailsViewController {
+    func reminderDateChanged(selectedDate: Date?) {
+        guard let selectedDate = selectedDate, let assignment = assignment.first else { return }
+        NotificationManager.shared.setReminder(for: assignment, at: selectedDate, studentID: studentID) { error in performUIUpdate {
+            if error == nil {
+                self.reminderDateButton.setTitle(selectedDate.dateTimeString, for: .normal)
```

Comment:
We could strongly capture `self` in the block's capture list so we could delete all `self.` references from this block.

Reply:
captured self.